views/index.hbs

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/style.css" />
+</head>
+
+<body>
+
+        <div class = container>
+            <div class="background-img">
+                <form action="/artist-search" method="GET">
+                    <input type="text" id="search" name="search" value=" "><br>
+                    <input type="submit" id="submit" value="Search for an Artist">
+                </form>   
+            </div>
+            
+        </div>
+
+
+</body>
+</html>
```

Comment:
In this file, you only need to have the `container div`. Layout.hbs is the default hbs that is rendered and then whatever you pass to res.render() will appear inside the {{{ body }}} in the layout.hbs.